cdci_data_analysis/pytest_fixtures.py

```diff
@@ -362,6 +362,36 @@ def dispatcher_test_conf_fn(tmpdir):
 
     yield fn
 
+@pytest.fixture
+def dispatcher_test_conf_empty_sentry_fn(tmpdir):
+    fn = os.path.join(tmpdir, "test-dispatcher-conf.yaml")
+    with open(fn, "w") as f:
+        f.write("""
+dispatcher:
+    dummy_cache: dummy-cache
+    products_url: PRODUCTS_URL
+    dispatcher_callback_url_base: http://localhost:8001
+    sentry_url: 
+    logstash_host: 
+    logstash_port: 
+    secret_key: 'secretkey_test'
+    bind_options:
+        bind_host: 0.0.0.0
+        bind_port: 8001
+    email_options:
+        smtp_server: 'localhost'
+        sender_email_address: 'team@odahub.io'
+        cc_receivers_email_addresses: ['team@odahub.io']
+        bcc_receivers_email_addresses: ['teamBcc@odahub.io']
+        smtp_port: 61025
+        smtp_server_password: ''
+        email_sending_timeout: True
+        email_sending_timeout_default_threshold: 1800
```

Comment:
This an be avoided by simply removing removing the `sentry_url` value from the previous one, or maybe with a parameter that controls the value of `sentry_url`. No need to duplicate.